example/an_app/config/initializers/cypress_rails_initializer.rb

```diff
@@ -8,8 +8,11 @@
 end
 
 CypressRails.hooks.after_server_start do
-  # After the server has  booted we add the compliment!
-  puts "Congrats the server has started!"
+  # After the server has  booted we add the compliment to the existing fixture list!
+  Compliment.create(text: "This shall be the first.")
+  if Compliment.count == 4
+    raise "I cannot run tests without compliments!"
```

Comment:
lol the random model i chose for this example app never fails to make me chuckle. nice one @dflynn15